### ISSUE TITLE
range: failing validation does not set error message

### ISSUE THREAD

**Issue by BernhardDenner** [ElektraInitiative/libelektra#1428]
I'm using the range plugin with a single range (didn't try multiple ranges). When trying to set an invalid value for the desired key, the range plugin fails but does not set an error message on the parent key.

e.g:
```sh
$> kdb set user/test/puppet-val/spec/x1 90
The command kdb set failed while accessing the key database with the info:

$> 
```
Seems to be an easy fix, since the error message is only set if the checking function return -1 but actually range fails return 0

**Comment by tom-wa** [comment]
thanks for reporting, #1429 fixes that issue

**Comment by markus2330** [comment]
Thank you for reporting!, I merged #1429 
@BernhardDenner Can you please close the issue if it is fixed?

**Comment by BernhardDenner** [comment]
fixed
